src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger LOGGER = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        LOGGER.info("Method login was called. Params: login={}", login);
```

Comment:
The log message should be more informative. Include both parameters 'login' and 'password' to make the log output more useful for debugging purposes. However, be cautious with logging sensitive information like passwords in plain text.

Reply:
---